ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -112,25 +140,36 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal {
         case alwaysHidden
     }
 
-    public typealias TokenSetKeyType = EmptyTokenSet.Tokens
-    public var tokenSet: EmptyTokenSet = .init()
+    public typealias TokenSetKeyType = NavigationBarTokenSet.Tokens
+    public lazy var tokenSet: NavigationBarTokenSet = .init(style: { [weak self] in
+        self?.style ?? NavigationBar.defaultStyle
+    })
 
     static let expansionContractionAnimationDuration: TimeInterval = 0.1 // the interval over which the expansion/contraction animations occur
 
     private static var defaultStyle: Style = .primary
 
+    // These two constants are based on OS default values
+    static let systemHeight: CGFloat = 44
```

Comment:
Shouldn't these constants live in `NavigationBarTokenSet`?

Reply:
These constants aren't really related to tokens; they're more closely related to the OS.

Reply:
Moved along with the other constants.